user_notification/tests.py

```diff
@@ -0,0 +1,22 @@
+from user_notification.models import JobAlert
+import pytest
+from datetime import datetime
+
+
+# test for calc_check_day
+@pytest.mark.parametrize('n, x, expected', [
+    ("2020-12-27", 2, "2020-12-29"),
+    ("2020-12-27", 3, "2020-12-30"),
+])
+def testCalc_check_date(n, x, expected):
+    assert JobAlert.calc_check_date(
+        n, x) == datetime.strptime(expected, "%Y-%m-%d").date()
+
+
+@pytest.mark.parametrize('n, x, expected', [
+    ("2020-12-27", 2, "2020-12-30"),
+    ("2020-12-27", 3, "2020-12-29"),
+])
+def testCalc_check_date2(n, x, expected):
```

Comment:
What is the different between those two tests?

Reply:
just wanted to make sure the calculation really works by testing wrong answers too
should I delete the second test?

Reply:
Yes... but maybe add a test of failure cases (for example negative numbers, invalid date, etc...

Reply:
something like that:
with pytest.raises(ValueError, ...) ?

Reply:
is it possible to approve this test for now and I'll try to learn how to make a test of failure cases? 
I've deleted the 2nd test that was unnecessary

Reply:
Better remove that test...

Reply:
I saw you removed it...